app/src/common/shared/com/igalia/wolvic/VRBrowserApplication.java

```diff
@@ -58,7 +58,7 @@ protected void onActivityCreate(@NonNull Context activityContext) {
         mConnectivityManager.init();
         mPlaces = new Places(activityContext);
         mServices = new Services(activityContext, mPlaces);
-        mLoginStorage = new LoginStorage(this);
+        mLoginStorage = new LoginStorage(activityContext);
```

Comment:
Hmm this seems unrelated although it looks like it's correct. Should we land it separatedely?

Reply:
The creation of `LoginStorage` is the first time that `EngineProvider.getOrCreateRuntime()` is called on startup. We need to pass the activity Context so it can use the methods in `WidgetManagerDelegate`.